### PR TITLE
Reverts sound range and falloff  so as to not carry as far

### PR DESCRIPTION
**Pull request by Robustin**
:cl: Robustin
del: Reverted changes in 3d sound system that tripled the distance that sound would carry. 
/:cl:

Apparently Fox's 3d Sound PR just threw a *3 into the sound distance formula. I removed the 3.

I'm open to further changes since I don't really understand sound code but having people literally hear what's going on in other departments and entire maint sections from the center of their own department was stupid.

### CODE REVIEW

**Comment by optimumtact** [comment]
was [changes sound range and falloff to not carry as far]

to complex a title for you

**Comment by ZeWaka** [comment]
I like how far sound caries, honestly. However, I think it could be dampened a bit. Maybe a magicnumber like 1.5 or 2 would be better than 3.

**Comment by KorPhaeron** [comment]
Good change. Heavily discourages stealth when any action can be heard from 30 tiles away

File: code/__DEFINES/sound.dm [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 
 #define SOUND_MINIMUM_PRESSURE 10
-#define FALLOFF_SOUNDS 0.5
+#define FALLOFF_SOUNDS 1
```
Comment (kevinz000) [inline code/__DEFINES/sound.dm]: you seem to have done more than just remove a *3, robustin.

**Comment by Fox-McCloud** [comment]
The whole point of 3D sounds is to add dimensionality to it in addition to long range. 

It does impact some aspects of stealth---but really only what I call "loud stealth".

That is to say, running into an empty room and eswording someone to death.

True stealth (like with ebows, silencing mixes, jammers, etc) are still quiet and typically only detectable in adjacent rooms.

Making stealth oriented weapons actually fit that niche is a *good* thing; having loud weapons also work well as stealth weapons in isolation? That just makes them even stronger, which I object to as a good design practice.

**Comment by Robustin** [comment]
@Fox-McCloud 

I think we're all cool with the idea of encouraging stealth but the massive distance increase made nothing stealthy. It just boils down to:

1) Even "loud" objects are too loud. Eswords, gunshots, their sound should carry pretty far but being able to hear them in another department was just too much. It was frustrating for antags because even secluded kills would be detected but it was annoying for non-antags who were getting bombarded with noises irrelevant to them. The classic example was hearing a door getting hacked only to find out the hacked door wasn't even in your department. 

2) There was no special treatment given to "stealth noises". Noises that should be "stealthy" were still largely indistinguishable in the code, which is why we've got issues like Clock Cult stealth being impossible because they have a lot of unique noises that can be heard from really, really far away. 

Once I fully appreciate how sound carries (as of writing this I just addressed a merge conflict that added ANOTHER sound var) I might tweak the "max distance" but with Bagil hitting 90 players regularly I'm already concerned about stealth being dead and am wary about a sound system that reinforces that trend.

**Comment by calzilla1** [comment]
Never trust a furry with your code

**Pull request merged**